src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexCultureTests.cs

```diff
@@ -117,12 +117,7 @@ public void TurkishI_Is_Differently_LowerUpperCased_In_Turkish_Culture(int lengt
 
             // same input and regex does match so far so good
             Assert.All(cultInvariantRegex, rex => Assert.True(rex.IsMatch(input)));
```

Comment:
worth repeating this test for `az` ?

Reply:
I have now added AZ coverage to my ignore test (outerloop) so that we have coverage for that too and so that we match all cases for that culture as well.